pinot-integration-test-base/src/test/java/org/apache/pinot/integration/tests/ClusterTest.java

```diff
@@ -84,27 +74,158 @@
 
 /**
  * Base class for integration tests that involve a complete Pinot cluster.
+ *
+ * Similar to {@link ControllerTest}, one should entirely encapsulate the {@code ClusterTest} object as a
+ * singleton within a test group/suite; or instantiate one instance per test class that requires setup.
  */
 public abstract class ClusterTest extends ControllerTest {
```

Comment:
I think we can rename it to `DefaultTestCluster` and make it a regular class instead of abstract class

Reply:
agreed. traditionally `Test*` and `*Test` class names are actually test cases. so we shouldn't use these as start or end class name for utilities.
see: https://maven.apache.org/surefire/maven-surefire-plugin/examples/inclusion-exclusion.html. I will follow up